test/jpypetest/test_startup.py

```diff
@@ -18,23 +18,20 @@
 import jpype
 import common
 import subrun
+import functools
 import os
+from pathlib import Path
 import sys
 import unittest
 
 
-def runStartJVM(*args, **kwargs):
-    jpype.startJVM(*args, **kwargs)
-
-
+@functools.wraps(jpype.startJVM)
```

Comment:
Only reason for this: So that I had the full signature of ``startJVM`` in my IDE (since it is no longer a catch-all :tada:)